### ISSUE TITLE
Support for biblatex

### ISSUE THREAD

**Issue by mowgli** [vim-latex/vim-latex#171]
biblatex is much more mature than the plain bibtex implementation.

Would it be possible to support biblatex (command BBB and such)?

I would also like to be able to customize the entry types completely. Maybe allowing some templates for bib(La)TeX entries.

**Comment by gerw** [comment]
What do you mean by `support biblatex`? Biblatex uses still `bib`-files and `BBB` does work).

**Comment by mowgli** [comment]
True. But it has other templates (for example @online) and more fields.

**Comment by gerw** [comment]
Indeed, feel free to contribute!

Btw: I do not write my bib files per hand, I mainly use https://github.com/gerw/mathbin/blob/master/arxiv2bibtex and https://github.com/gerw/mathbin/blob/master/doi2bibtex.
